lib/ruby-debug-ide/command.rb

```diff
@@ -118,7 +118,7 @@ def timeout(sec)
     def debug_eval(str, b = get_binding)
       begin
         str = str.to_s
-        str.force_encoding('UTF-8') if(RUBY_VERSION > 2.0)
+        str.force_encoding('UTF-8') if(RUBY_VERSION > '2.0')
```

Comment:
This change does not relate to the issue one is fixing at the moment. Please revert this chunk and commit in a separate PR